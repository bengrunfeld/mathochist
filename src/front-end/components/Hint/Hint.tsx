import React from 'react'

const Hint = () => (
  <div>
    <h1>Hint</h1>
  </div>
)

export default Hint
