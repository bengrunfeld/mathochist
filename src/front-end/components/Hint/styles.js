import s from 'styled-components'

const HintContainer = s.div`
  background: #cfe1f3;
`

const HintText = s.p`
  padding: 6px;
  font-size: 17px;
  color: #434343;
  text-align: center;
`

export { HintContainer, HintText }
