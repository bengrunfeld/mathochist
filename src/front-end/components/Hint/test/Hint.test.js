import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import Hint from '../Hint'

describe('<Hint />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<Hint />)
    expect(tree).toMatchSnapshot()
  })
})
