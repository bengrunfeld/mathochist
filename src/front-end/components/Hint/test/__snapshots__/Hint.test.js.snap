// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hint /> matches the snapshot 1`] = `
<Hint>
  <div>
    <h1>
      Hint
    </h1>
  </div>
</Hint>
`;
